webapp/components/suggestion/switch_channel_provider.jsx

```diff
@@ -57,83 +67,179 @@ class SwitchChannelSuggestion extends Suggestion {
     }
 }
 
+let prefix = '';
```

Comment:
It's only used in this case and is used in a function outside of the provider, if you're 4/5 or 5/5 on this I can change it but otherwise I'd don't think it's a big deal